docs/content/latest/quick-start/build-apps/java/ysql-yb-jdbc.md

```diff
@@ -0,0 +1,354 @@
+          + "&topology-keys=aws.us-west.us-west-2a";
+
+       try {
+          for(int i=0; i<6; i++) {
+             Connection connection = DriverManager.getConnection(yburl);
+             connectionList.add(connection);
+          }
+
+          System.out.println("You can verify the load balancing by visiting http://<host>:13000/rpcz as discussed before");
+          System.out.println("Enter a integer to continue once verified:");
+          int x = new Scanner(System.in).nextInt();
+
+          System.out.println("Closing the connections!!");
+          for(Connection connection : connectionList) {
+             connection.close();
+          }
+
+        }
+       catch (SQLException exception) {
+          exception.printStackTrace();
+       }
+
+    }
+
+    public static void makeConnectionUsingYbClusterAwareDataSource() {
+       System.out.println("Now, Lets create 10 connections using YbClusterAwareDataSource and Hikari Pool");
+
+       Properties poolProperties = new Properties();
+       poolProperties.setProperty("dataSourceClassName", "com.yugabyte.ysql.YBClusterAwareDataSource");
+       //the pool will create  10 connections to the servers
+       poolProperties.setProperty("maximumPoolSize", String.valueOf(10));
+       poolProperties.setProperty("dataSource.serverName", "127.0.0.1");
+       poolProperties.setProperty("dataSource.portNumber", "5433");
+       poolProperties.setProperty("dataSource.databaseName", "yugabyte");
+       poolProperties.setProperty("dataSource.user", "yugabyte");
+       poolProperties.setProperty("dataSource.password", "yugabyte");
+       // If you want to provide additional end points
+       String additionalEndpoints = "127.0.0.2:5433,127.0.0.3:5433";
+       poolProperties.setProperty("dataSource.additionalEndpoints", additionalEndpoints);
+
+       // If you want to load balance between specific geo locations using topology keys
+       String geoLocations = "aws.us-west.us-west-2a";
+       poolProperties.setProperty("dataSource.topologyKeys", geoLocations);
+
+
+       HikariConfig config = new HikariConfig(poolProperties);
+       config.validate();
+       HikariDataSource hikariDataSource = new HikariDataSource(config);
+
+       System.out.println("Wait for some time for Hikari Pool to setup and create the connections...");
+       System.out.println("You can verify the load balancing by visiting http://<host>:13000/rpcz as discussed before.");
+       System.out.println("Enter a integer to continue once verified:");
+       int x = new Scanner(System.in).nextInt();
+
+       System.out.println("Closing the Hikari Connection Pool!!");
+       hikariDataSource.close();
+
+    }
+  
+  }
+  ```
+
+  {{< note title="Note">}}
+
+   When using `DriverManager.getConnection()`, you need to include the `load-balance=true` property in the connection URL. In the case of `YBClusterAwareDataSource`, load balancing is enabled by default but you must set property `dataSource.topologyKeys`.
```

Comment:
load balancing is enabled by default, but you must set the `dataSource.topologyKeys` property.